lms/services/__init__.py

```diff
@@ -31,7 +31,7 @@
 
 def includeme(config):
     config.register_service_factory(
-        "lms.services.application_instance_getter.ApplicationInstanceGetter",
+        "lms.services.application_instance_getter.application_instance_getter_service_factory",
```

Comment:
I know it's not current practice, but can you just put the actual function / a lambda here? Or does that break something to do with the design where you might not even import the code etc?

Imports vs. strings are nice, as the code won't even compile if you change the name on one side etc.

Reply:
I'm not sure whether pyramid_services accepts callables directly, or whether it only works with strings.

I know that the authors prefer passing strings to this and other Pyramid registries because it's easier because the registering code doesn't even have to import the thing it's registering